enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -1247,7 +1247,7 @@ func downloadLayer(ctx context.Context, layer ctr.Layer, destDir string) error {
 	defer os.RemoveAll(tempUnpackDir)
 
 	// TODO: avoid tar command.
-	cmd := exec.CommandContext(ctx, "tar", "--no-same-owner", "--extract", "--directory", tempUnpackDir)
+	cmd := exec.CommandContext(ctx, "tar", "--extract", "--directory", tempUnpackDir)
```

Comment:
This is the only non-test change in this PR (removing `--no-same-owner`). This flag was causing permissions errors when running as non-root, but a while back I decided not to support non-root for `ociruntime` for now, since it adds a lot of complexity.